spec/features/admin/product_import_spec.rb

```diff
@@ -504,32 +504,21 @@
   end
 
   describe "handling a large file (120 data rows)" do
-    let!(:producer) { enterprise }
-
-    let(:tmp_csv_path) { "/tmp/test.csv" }
+    let!(:producer) {enterprise}
+    let!(:tax_category) { create(:tax_category, name: "Tax Category Name") }
+    let!(:shipping_category) { create(:shipping_category, name: "Shipping Category Name") }
+    
+    let!(:csv_file) { file_fixture('sample_file_120_products.csv') }
 
     before do
       login_as admin
       visit main_app.admin_product_import_path
     end
 
-    context "when importing to product list" do
-      def write_tmp_csv_file
-        CSV.open(tmp_csv_path, "w") do |csv|
-          csv << ["name", "producer", "category", "on_hand", "price", "units", "unit_type",
-                  "tax_category", "shipping_category"]
-          120.times do |i|
-            csv << ["Imported Product #{i + 1}", producer.name, category.name, 1, "1.00", "500",
-                    "g", tax_category.name, shipping_category.name]
-          end
-        end
-      end
-
-      before { write_tmp_csv_file }
-
-      it "validates and saves all batches" do
+    context "from a fixture path" do
```

Comment:
I think this puts the focus on the way the test is implemented rather than on the actual logic under test. IMO this context description shouldn't change.

Reply:
Totally agree: I've reverted the description changes on the `context` and `it` blocks then squashed the commits.